doc/sphinxman/source/glossary_psivariables.rst

```diff
@@ -79,20 +79,155 @@ PSI Variables by Alpha
    for the averaged coupled-pair functional level of theory.
 
 .. psivar:: ADC ROOT 0 -> ROOT m EXCITATION ENERGY
-   ADC ROOT 0 (h) -> ROOT m (i) EXCITATION ENERGY
-   ADC ROOT 0 -> ROOT m EXCITATION ENERGY - h TRANSITION
+   TD-fctl ROOT 0 -> ROOT m EXCITATION ENERGY
+
+   The excitation energy from ground state to root *m*.
+   DFT functional labeled if canonical.
+
+.. psivar:: ADC ROOT 0 (IN h) -> ROOT m (IN i) EXCITATION ENERGY
+   TD-fctl ROOT 0 (IN h) -> ROOT m (IN i) EXCITATION ENERGY
+
+   The excitation energy from the ground state (which is of irrep *h*)
```

Comment:
At some point (needn't be before merge), `[Eh]` and other units should be added.